README.md

```diff
@@ -1,3 +1,74 @@
-<h1 align="center">
-    Inside
-</h1>
+# Inside
+
+Inside is an open source platform to deal with all the intern process of a company related to its employees. Our goal is to make easy to keep a track and manage all information related to the people that work **inside** the company.
+
+## Getting Started
+
+To get started you will need to get a copy of the project, insert the following commands (one by one) on your terminal.
+
+```javascript
+git clone git@github.com:ifactory-solutions/inside-client.git
+cd inside-client
+npm install
+```
+
+You will get a clone of the project, navigate to it, and install its depencies.
+
+The project was initiated with the awesome facebook's cli for react (create-react-app). So you have all its default commands available, like:
+
+```
+npm run build
+npm run test
+npm start
+```
+
+### Prerequisites
+
+You should have at least the LTS node version 6.11. We recommend you to use [nvm](https://github.com/creationix/nvm) it will make easy for you to switch between node/mpm versions.
+
+
+### Installing
+
+Run `npm start` (into the project folder) to run the application over a local server.
+
+```
+cd inside-client
+npm start
+```
+
+## Built With
```

Comment:
I think this section is not correct, we have some incorrect links (Apollo client, Styled Components, Jest) and some incorrect descriptions

Reply:
Thanks for figuring this out. My clipboard must have gone crazy. I fixed, can you check if everything is ok now?